smsdk/client.py

```diff
@@ -702,17 +805,21 @@ def get_machine_type_names(self, clean_strings_out=True):
         else:
             return machine_types["source_type"].unique().tolist()
 
-    def get_raw_data(
-        self,
-        raw_data_table,
-        normalize=True,
-        fields=[],
-        time_selection=one_day_relative,
-        limit=400,
-        offset=0,
-        *args,
-        **kwargs,
-    ):
+    def get_raw_data(self, *args, normalize=True, **kwargs):
+        raw_data_table = kwargs.pop("raw_data_table", None)
+        if raw_data_table is None and args:
+            raw_data_table = args[0]
+            args = args[1:]
+
+        fields = kwargs.pop("fields", None)
+        if fields is None and args:
+            fields = args[0]
+            args = args[1:]
+        elif fields is None:
+            fields = []
+
+        time_selection = kwargs.pop("time_selection", ONE_DAY_RELATIVE)
```

Comment:
Time_selection, limit an offset are all unpassable via args here

Reply:
Fixed the issue.